src/main/java/org/folio/dataexp/service/ConsortiaService.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.dataexp.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.dataexp.client.ConsortiaClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class ConsortiaService {
+
+  @Value("${myuniversity}")
+  private String tenantId;
```

Comment:
tenantId will be not found in multiple tenant environment, to get tenant information please use FolioExecutionContext bean.

Reply:
Good catch, thank you.